deeplake/core/tensor.py

```diff
@@ -1823,8 +1922,6 @@ def create_vdb_index(
 
     def delete_vdb_index(self, id: str):
         self.storage.check_readonly()
-        if self.meta.htype != "embedding":
```

Comment:
add validation if the tensor is embedding or text tensor.

Reply:
Done.